irc.md

```diff
@@ -9,10 +9,10 @@ showinnav: true
 
 While traditionally the best way to communicate in AHA was the [mailing
 list]({{ site.mailinglistpage }}), now the most common place to find us
-is on IRC. We can be found on the [Freenode](http://freenode.net/) IRC
+is on IRC and discord. We can be found on the [Libera](http://libera.chat/) IRC
 Network in the channel #AHA.
 
-* Server: chat.freenode.net
+* Server: irc.libera.chat
```

Comment:
I believe Libera network requires SSL, so this should be updated to either append :6697 (+SSL) or as a second line to make that clear for people